packages/tizen_rpc_port/tizen/src/tizen_rpc_port_plugin.cc

```diff
@@ -0,0 +1,846 @@
+      T &out) {
+    if (std::holds_alternative<flutter::EncodableMap>(*args)) {
+      flutter::EncodableMap map = std::get<flutter::EncodableMap>(*args);
+      if (map.find(flutter::EncodableValue(key)) != map.end()) {
+        flutter::EncodableValue value = map[flutter::EncodableValue(key)];
+        if (std::holds_alternative<T>(value)) {
+          out = std::get<T>(value);
+          return true;
+        }
+      }
+      LOG_DEBUG("Key %s not found", key);
+    }
+
+    return false;
+  }
+
+  bool GetEncodableValueFromArgs(const flutter::EncodableValue *args,
+      const char *key, flutter::EncodableValue &out) {
+    if (std::holds_alternative<flutter::EncodableMap>(*args)) {
+      flutter::EncodableMap map = std::get<flutter::EncodableMap>(*args);
+      if (map.find(flutter::EncodableValue(key)) != map.end()) {
+        out = map[flutter::EncodableValue(key)];
+        return true;
+      }
+    }
+
+    return false;
+  }
+
+  void HandleMethodCall(
+      const flutter::MethodCall<flutter::EncodableValue> &method_call,
+      std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> result) {
+    const auto &method_name = method_call.method_name();
+    auto found = method_handlers_.find(method_name);
+    if (found == method_handlers_.end()) {
+      LOG_ERROR("%s it not implmeneted", method_name.c_str());
+      result->NotImplemented();
+      return;
+    }
+
+    LOG_DEBUG("HandleMethodCall. method_name: %s", method_name.c_str());
+    auto method = found->second;
+    const flutter::EncodableValue* args = method_call.arguments();
+    method(args, std::move(result));
+  }
+
+  void StubCreate(const flutter::EncodableValue* args,
+      std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> result) {
+    LOG_DEBUG("StubCreate");
+    std::string port_name;
+    if (!GetValueFromArgs<std::string>(args, "portName", port_name)) {
+      result->Error("Invalid parameter");
+      return;
+    }
+
+    auto found = native_stubs_.find(port_name);
+    if (found != native_stubs_.end()) {
+      result->Error("Already exists");
+      return;
+    }
+
+    native_stubs_[port_name] = std::make_shared<tizen::RpcPortStub>(port_name);
+    result->Success();
+  }
+
+  void StubDestroy(const flutter::EncodableValue* args,
+      std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> result) {
+    LOG_DEBUG("Destroy");
+    std::string port_name;
+    if (!GetValueFromArgs<std::string>(args, "portName", port_name)) {
+      result->Error("Invalid parameter");
+      return;
+    }
+
+    {
+      auto found = event_channels_.find(port_name);
+      if (found != event_channels_.end())
+        event_channels_.erase(found);
+    }
+
+    auto found = native_stubs_.find(port_name);
+    if (found == native_stubs_.end()) {
+      result->Error("Invalid parameter");
+      return;
+    }
+
+    native_stubs_.erase(found);
+    result->Success();
+  }
+
+  void SetTrusted(const flutter::EncodableValue* args,
```

Comment:
Please change this name to 'StubSetTrusted'.
This method is for the stub.

Reply:
I changed the method name. Thanks :)